.github/workflows/test.yml

```diff
@@ -26,6 +26,15 @@ jobs:
       - uses: astral-sh/setup-uv@v4
         with:
           version: ">=0.5.0"
+          enable-cache: true
+
+      - name: Format toml
+        run: uvx taplo fmt
```

Comment:
Does this format or crash if the format is not correct (it should be the latter imo)

Reply:
@domoritz I'm actually not sure - I'll test it now so we can find out what `taplo` does.

This is what the `ruff` one looks like on fail: [commit](https://github.com/vega/vega-datasets/pull/647/commits/3706ebb9c9907fb9faf4d905e2dbfc320dd33cfc), [failure](https://github.com/vega/vega-datasets/actions/runs/12330436827/job/34415906150?pr=647)

Reply:
@domoritz thanks for flagging this!

I've corrected the action, such that this is now what you'll see on a failure:

https://github.com/vega/vega-datasets/actions/runs/12341076765/job/34439287045?pr=647